docs/recipes/basic-transactions.md

```diff
@@ -68,4 +68,30 @@ and account, check out [calculating rent exemption](accounts.md#calculating-rent
 @[code](@/code/basic-transactions/calc-tx-cost/calc-tx-cost.en.ts)
 
   </CodeGroupItem>
-</CodeGroup>
\ No newline at end of file
+</CodeGroup>
+
+## Adding memo
+
+Any transaction can add a message making use of the [memo program][2].
+
+<CodeGroup>
+  <CodeGroupItem title="TS" active>
+
+@[code](@/code/basic-transactions/memo/memo.en.ts)
+
+  </CodeGroupItem>
+  <CodeGroupItem title="Wallet-Adapter">
+
+@[code](@/code/basic-transactions/sending-sol/sending-sol.adapter.en.tsx)
+
+  </CodeGroupItem>
+  <CodeGroupItem title="CLI">
+
+@[code](@/code/basic-transactions/memo/memo.en.sh)
+
+  </CodeGroupItem>
+</CodeGroup>
+
+Currently the `programID` from the **Memo Program** has to be added manually `MemoSq4gqABAXKb96qnH8TysNcWxMyWCqXgDLGmfcHr`.
```

Comment:
Could we have this up top?
So it would be like: 

Any transaction can add a message making use of the [memo program][2]. Currently the `programID` from the **Memo Program** has to be added manually `MemoSq4gqABAXKb96qnH8TysNcWxMyWCqXgDLGmfcHr`.